inc/classes/busting/trait-file-busting.php

```diff
@@ -85,7 +85,7 @@ public function delete() {
 	 * @return string|bool Version of the file. False if the file does not exist.
 	 */
 	protected function get_busting_version() {
-		if ( isset( $this->file_version ) ) {
+		if ( isset( $this->file_version ) && empty( $this->file_version ) ) {
```

Comment:
Same here. We touched this file. Time to move it into the module and rename.

Reply:
Done.